.pre-commit-config.yaml

```diff
@@ -13,14 +13,14 @@ repos:
     hooks:
       - id: isort
         language_version: python3.11
-        args: ["--profile", "black", "--line-length=79"]
+        args: ["--profile", "black", "--line-length=100"]
```

Comment:
Dit werkt prima voor mensen met grote schermen, maar is dit niet net lang? Comments opknippen toch beter misschien

Reply:
Ruff standaard lijkt 88 te zijn, ik zal hem daar op zetten. Hij zal behoorlijk wat overhoop halen in de volgende commit denk ik